src/frontend/context/SimuleringContext.tsx

```diff
@@ -57,10 +63,71 @@ const [SimuleringProvider, useSimulering] = constate(({ åpenBehandling }: IProp
     const hentÅrISimuleringen = (perioder: ISimuleringPeriode[]): number[] =>
         [...new Set(perioder.map(periode => dayjs(periode.fom).year()))].sort();
 
+    const tilbakekrevingErToggletPå = toggles[ToggleNavn.tilbakekreving];
+
+    const erFeilutbetaling =
+        simuleringsresultat.status === RessursStatus.SUKSESS &&
+        simuleringsresultat.data.feilutbetaling > 0;
+
+    const tilbakekreving = useFelt<TilbakekrevingAlternativ | undefined>({
+        verdi: undefined,
+        avhengigheter: { tilbakekrevingErToggletPå, erFeilutbetaling },
+        skalFeltetVises: avhengigheter =>
+            avhengigheter?.tilbakekrevingErToggletPå && avhengigheter?.erFeilutbetaling,
+        valideringsfunksjon: felt =>
+            felt.verdi === undefined
+                ? feil(
+                      felt,
+                      'Resultatet medfører en feilutbetaling. Du må velge om det skal opprettes tilbakekrevingsbehandling.'
+                  )
+                : ok(felt),
+    });
+    const fritekstVarsel = useFelt<string>({
+        verdi: '',
+        avhengigheter: { tilbakekrevingErToggletPå, tilbakekreving, erFeilutbetaling },
+        valideringsfunksjon: (felt, avhengigheter) =>
+            avhengigheter?.erFeilutbetaling &&
+            avhengigheter?.tilbakekreving?.verdi === TilbakekrevingAlternativ.OPPRETT_SEND_VARSEL &&
+            felt.verdi === ''
+                ? feil(felt, 'Du må skrive en fritekst for varselet til tilbakekrevingen.')
+                : ok(felt),
+        skalFeltetVises: (avhengigheter: Avhengigheter) =>
+            avhengigheter?.tilbakekrevingErToggletPå &&
+            avhengigheter?.erFeilutbetaling &&
+            avhengigheter?.tilbakekreving?.verdi === TilbakekrevingAlternativ.OPPRETT_SEND_VARSEL,
+    });
+    const begrunnelse = useFelt<string>({
+        verdi: '',
+        avhengigheter: { erFeilutbetaling, tilbakekrevingErToggletPå },
+        skalFeltetVises: avhengigheter =>
+            avhengigheter?.tilbakekrevingErToggletPå && avhengigheter?.erFeilutbetaling,
+        valideringsfunksjon: felt =>
+            felt.verdi === ''
+                ? feil(felt, 'Du må skrive en begrunnelse for valget om tilbakekreving.')
+                : ok(felt),
+    });
+
+    const { skjema, hentFeilTilOppsummering, onSubmit } = useSkjema<
+        {
+            tilbakekreving: TilbakekrevingAlternativ | undefined;
+            fritekstVarsel: string;
+            begrunnelse: string;
+        },
+        IFagsak
+    >({
+        felter: { tilbakekreving, fritekstVarsel, begrunnelse },
+        skjemanavn: 'Du må skrive en bgrunnelse for tilbakekreving',
```

Comment:
Dette feltet brukes ikke for øyeblikket, men er ment som en en beskrivelse av skjemaet. Kanskje noe ala "opprett tilbakekreving" eller noe 🤷‍♂️